lib/spack/spack/cmd/info.py

```diff
@@ -186,8 +227,11 @@ def print_text_info(pkg):
                     inverse_map[when] = set()
                 inverse_map[when].add(spec)
         for when, specs in reversed(sorted(inverse_map.items())):
-            print("    %s provides %s" % (
-                when, ', '.join(str(s) for s in specs)))
+            line = "    %s provides %s" % (
+                when.colorized(), ', '.join(s.colorized() for s in specs)
```

Comment:
Just a warning: I had a couple of weird python failures related to the colorization of the provided specs at the end of `spack info`. The failures happened only on `python@3.3` and `python@3.4`, and I couldn't reproduce them locally. Restarting the jobs apparently made Travis go green. Let me know if, for safety, you prefer to remove colorization or if we are fine like this. Other than that this is for me ready to be merged.